### ISSUE TITLE
Add data of tests performed

### ISSUE THREAD

**Issue by jsulmer** [daenuprobst/covid19-cases-switzerland#65]
Recently BAG also publish data on how many patients get tested. It seem to be only a rough estimate, but if compared to daily reported cases, it will show a change in infection rate among the tested sub population (patients at risk showing symptoms), or at least a change in behaviour woh is going to test himself.

**Comment by flueko** [comment]
I agree totally: tests performed, percentage positive, hospitalized, intenisive care, hospital capacity.

**Comment by MatthieuStigler** [comment]
Can you share the link for that data?

**Comment by dominikgehl** [comment]
It's part of the openZH files in the ncumul_tested column.

**Comment by dominikgehl** [comment]
see https://github.com/daenuprobst/covid19-cases-switzerland/pull/68
